docs/accounts/balance/get_balance_for_account.md

```diff
@@ -31,18 +31,23 @@ description: Get the current balance for a given account.
 {
   "method": "get_balance_for_account",
   "result": {
-    "balance": {
-      "object": "balance",
-      "network_block_height": "152918",
-      "local_block_height": "152918",
-      "account_block_height": "152003",
-      "is_synced": false,
-      "unspent_pmob": "110000000000000000",
-      "max_spendable_pmob": "110000000000000000",
-      "pending_pmob": "0",
-      "spent_pmob": "0",
-      "secreted_pmob": "0",
-      "orphaned_pmob": "0"
+    "balance_per_token": {
+      "0": {
```

Comment:
It might be nice to prefer the enum label rather than the int value here, especially as the number of tokens grows and the ints have less semantic meaning